### ISSUE TITLE
warning: char subscripts / use unsigned char

### ISSUE THREAD

**Issue by tom-wa** [ElektraInitiative/libelektra#939]
```
/home/thomas/libelektra/src/libs/ease/array.c:63:3: warning: array subscript has type 'char' [-Wchar-subscripts]
   for (; isdigit (*current); current++)
   ^
```

```
-Wchar-subscripts
    Warn if an array subscript has type char. This is a common cause of error, as programmers often forget that this type is signed on some machines. This warning is enabled by -Wall. 
```

**Comment by markus2330** [comment]
Should be an easy fix?

**Comment by e1528532** [comment]
fixed in #1508
